src/pages/builders.js

```diff
@@ -5,7 +5,7 @@ import { Button } from '@newrelic/gatsby-theme-newrelic';
 import PageLayout from '../components/PageLayout';
 import ExternalLink from '../components/ExternalLink';
 import FeatherIcon from '../components/FeatherIcon';
-import devReliquary from '../images/builders/The_Relicans.png';
+import devReliquary from '../images/builders/relicans.png';
```

Comment:
Was there an image at the old path? If its not used anymore, should we delete that image?

Reply:
Yes, i believe I have deleted the image already, but I will double check.